<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SatMapDialog</class>
 <widget class="QDialog" name="SatMapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1860</width>
    <height>900</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Tracking map</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonRefresh">
       <property name="text">
        <string>Actualiser</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelStepTimeName">
       <property name="text">
        <string>Pas de temps</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxStepTime">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelStepTimeUnit">
       <property name="text">
        <string>sec</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTimeMarker">
       <property name="text">
        <string>Marque de temps</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxMarker">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTimeMarkerUnit">
       <property name="text">
        <string>h</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelDurationName">
       <property name="text">
        <string>Dur√©e</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxDuration">
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDurationHourUnit">
       <property name="text">
        <string>h</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonTrack">
       <property name="text">
        <string>Track</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="text">
        <string>Fermer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
